apps/flows/src/app/transaction-details-flow/for-you/for-you.component.scss

```diff
@@ -2,3 +2,71 @@
   position: absolute;
   inset: 0;
 }
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-gap: 24px;
+  gap: 24px;
+}
+
+/* Let grid items span all columns by default */
+.grid-item {
+  grid-column: span 12;
+}
+
+/* Tablet size and above */
+@media (min-width: 721px) {
+  .half-at-tablet-up {
+    grid-column: span 6;
+  }
+}
+
+/* Desktop size and above */
+@media (min-width: 1025px) {
+  .half-at-desktop-up {
+    grid-column: span 6;
+  }
+
+  .third-at-desktop-up {
+    grid-column: span 4;
+  }
+}
+
+/*********************************************************
+  Additional styles that are not essential to the example
+ *********************************************************/
+main {
+  background-color: var(--kirby-background-color);
+  min-height: 100%;
+}
+
+.max-width-container {
+  max-width: 1196px;
+  margin-inline: auto;
+}
+
+.gutter {
+  padding-inline: 16px;
+}
+
+.safe-area-inline {
+  padding-inline: 16px;
+}
+
+.box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  min-height: 200px;
+}
+
+.align-center {
+  text-align: center;
+}
+
+.right {
+  position: relative;
+  float: right;
+}
```

Comment:
Be careful when you copy code from somewhere else. You should remove styles that are not used for anything and make sure you actually want/need the remaining styles. E.g., you probably don't want the `main` style rule when the `main` element is not part of your template.

Reply:
Main is removed